control/control.CAASP.xml

```diff
@@ -437,31 +354,78 @@ configured accordingly to match the use case of the role. &lt;/p&gt;</label>
           <label>Administration Node (Dashboard)</label>
         </dashboard_role>
         <dashboard_role_description>
-          <label>• A set of tools and a dashboard for managing cluster nodes.</label>
-        </dashboard_role_description>
+          <label>• MicroOS plus CaaS Platform Kubernetes Stack
+• Provides Velum administration dashboard
+• Should be installed before Cluster Nodes</label>
+	</dashboard_role_description>
         <worker_role>
           <!-- TRANSLATORS: a label for a system role -->
           <label>Cluster Node</label>
         </worker_role>
         <worker_role_description>
-          <label>• The system is registered in a given administration dashboard.</label>
+          <label>• MicroOS plus CaaS Platform Kubernetes Stack
+• Configured as a cluster worker node
+• Should be installed after the Admin/Dashboard Node</label>
         </worker_role_description>
         <none_role>
           <!-- TRANSLATORS: a label for a system role -->
           <label>Plain System</label>
         </none_role>
         <none_role_description>
-          <label>• No services started by default.</label>
+          <label>• MicroOS plus an unconfigured CaaS Platform Kubernetes Stack</label>
         </none_role_description>
-        <micro_os_role>
-          <!-- TRANSLATORS: a label for a system role -->
-          <label>SUSE MicroOS</label>
-        </micro_os_role>
-        <micro_os_role_description>
-          <label>• No services started by default.</label>
-        </micro_os_role_description>
     </texts>
 
+    <proposals config:type="list">
+        <proposal>
+            <label>Installation Settings</label>
+            <mode>installation</mode>
+            <stage>initial</stage>
+            <name>initial</name>
+            <unique_id>inst_initial</unique_id>
+            <enable_skip>no</enable_skip>
+            <proposal_modules config:type="list">
+                <proposal_module>
+                    <name>bootloader</name>
+                    <presentation_order>40</presentation_order>
+                </proposal_module>
+                <proposal_module>
+                    <name>hwinfo</name>
+                    <presentation_order>80</presentation_order>
+                </proposal_module>
+                <proposal_module>
+                    <name>partitions</name>
+                    <presentation_order>10</presentation_order>
+                </proposal_module>
+                <!-- software proposal should be computed almost at the end -->
```

Comment:
I worry that this comment and reality is not in sync. But software should be definitively after timezone, network and kdump as all of them may need to add to software proposal additional software. ( if it does not do that now, it is bug and we will fix it ).

Reply:
@thkukuk what do you think? You're the one who wanted software so high on the list

we haven't seen a problem with this on Kubic but I see exactly where @jreidinger is coming from

Reply:
well, you see mainly difference when you play with proposal later, then some software which is required can miss. And also please note again that it is not order which is used for displaying it ( presentation order is used ), only order in which proposals are computed. And other modules can say software proposal that it need something and that software should add it.

Reply:
As Josef wrote, display order is not compute order. Compute the software proposal as last step sounds correct to me, too.